frontend/src/components/CourseCartCard/CourseCartCard.tsx

```diff
@@ -23,6 +23,18 @@ const CourseCartCard = ({ code, title }: Props) => {
     dispatch(addTab(code));
   };
 
+  const handleConfirm = async () => {
+    try {
+      const res = axios.post(
+        `/removeCourse`,
+        JSON.stringify({courseCode: code})
+      );
```

Comment:
you'd need to get the token using `useSelector` and then pass that into the request